scripts/post-merge.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Copyright 2022 The Parca Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +
```

Comment:
Maybe we should move this to `make clean` here: https://github.com/parca-dev/parca/blob/a794316dc2a3cdc9e746d4de249a18c7069d2b93/Makefile#L44

And use it here? 

But not sure what's the best practice, shall get others' thoughts as well.

Reply:
I think that makes sense, there are other occasions where people might want to run this, and people also may not necessarily have `pre-commit` installed.

Reply:
okay then, I'll rework this to be in that file.